.travis.yml

```diff
@@ -14,11 +8,80 @@ before_install:
 
 script: bundle exec rake
 
-env:
-  - ADAPTER=active_record
-  - ADAPTER=mongoid MONGOID_VERSION=3
-  - ADAPTER=mongoid MONGOID_VERSION=4
-  - ADAPTER=mongoid MONGOID_VERSION=5
+matrix:
+  include:
+    - rvm: 2.2.3
```

Comment:
Let's change this to test 2.2.x (where x is the latest release of the 2.2 line), 2.1.x and 2.0.x?